tools/fix_auth.rb

```diff
@@ -13,6 +13,7 @@
 
 require 'active_support/all'
 require 'active_support/concern'
+require 'manageiq-gems-pending'
```

Comment:
@jrafanie this is a command wrapper that is not included in our suite - does not run our suite. So this is probably not tested.

We could create a separate suite to load this. (or we could split into a separate rep)